devDocs/githubPullRequestTemplateExplanationAndExamples.md

```diff
@@ -124,44 +124,43 @@ Most items in the checklist have a section in the PR template where you can add
 so may preempt questions from the reviewer ensuring you are on the same page, and speed up the
 review process.
 
-### Pull Request description is up to date.
-Authors must keep the PR description up to date.
-- Even if changes to the approach are described in the comments for the PR.
-- Future developers need a concise explanation of a change.
-After each modification, check that the PR description is still accurate.
-
-### Unit tests
-- Discuss under "testing strategy" heading.
-- Describe the coverage of automated unit tests?
-- Is the changed code already, or can it be covered by automated unit tests?
-
-### System tests
-- Discuss under "testing strategy" heading.
-- Describe the coverage of automated system tests?
-- Is the changed code already, or can it be covered by automated unit tests?
-
-### Manual tests
-- Discuss under "testing strategy" heading. 
-- How did you manually test the change?
-- Be clear on steps another user can take to replicate your testing.
-- Is the described manual testing appropriate for the change?
-- Clearly describing this helps alpha testers, and future developers.
-- As a reviewer, please use this description to replicate the testing (if possible).
+### Pull Request description:
+- description is up to date.
+  Authors must keep the PR description up to date.
+  - Even if changes to the approach are described in the comments for the PR.
+  - Future developers need a concise explanation of a change.
+  After each modification, check that the PR description is still accurate.
+- change log entries
+  Has an appropriate change log entry been supplied.
+  As a reviewer, please review it.
+
+### Testing
+- Unit tests
+  - Discuss under "testing strategy" heading.

```

Comment:
Would you be able to move this line (and other usages) under the common "Testing" heading.

Reply:
Done in 6e354947cd429d8fae5016ea5222db201de2b0b9.